rtv/docs.py

```diff
@@ -43,3 +43,17 @@
   `LEFT` or `h`       : Return to subreddit mode
   `RIGHT` or `l`      : Fold the selected comment, or load additional comments
 """
+
+COMMENT_FILE = """
+--% {0} %--
+Do not edit the line above.
+Type your comment above the line.
+All text below the line will be discarded.
+
+If there is nothing but whitespace above the line
+then the comment will be discarded.
+
+Replying to {1}'s {2}
+
+{3}
+"""
```

Comment:
They look a little clunky to me, building and passing fullblown classes, instead of a builtin string.

Also they're much slower (more than 10x slower)
```python
Python 3.4.3 (default, Feb 26 2015, 23:01:07) 
[GCC 4.9.2 20150204 (prerelease)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from string import Template
>>> from timeit import timeit
>>> t = Template('$test')
>>> def template_test():
...     t.substitute(test='hey')
... 
>>> def format_test():
...     '{0}'.format('hey')
... 
>>> timeit(stmt=template_test)
5.1952612300010514
>>> timeit(stmt=format_test)
0.4597018319982453
```

Can you give me some reasons as to why string.Template would be better?